app/assets/stylesheets/trips.css.scss

```diff
@@ -1,3 +1,12 @@
+#map{
+  width: 70%;
+  height: 400px;
+  margin-right: auto;
+  margin-left: auto;
+  border-radius: 5px 5px 5px 5px;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `5px` instead of `5px 5px 5px 5px`